modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -1447,4 +1448,20 @@ private static class Augmentation {
             this.addition = addition;
         }
     }
+
+    private static Map<String, Integer> extractDataNodes(Entry dataNodesEntry) {
+        if (!dataNodesEntry.empty()) {
+            return fromBytes(dataNodesEntry.value());
+        } else {
+            return emptyMap();
+        }
+    }
+
+    private static long extractChangeTriggerRevision(Entry revisionEntry) {
+        if (!revisionEntry.empty()) {
+            return bytesToLong(revisionEntry.value());
+        } else {
+            return 0;
```

Comment:
It's not clear, and thus a bit unreliable that zero here is the same zero you use in DistributionZoneUtils
`if (scaleUpTriggerRevision != 0)` Let's use constant, e.g. INITIAL_TRIGGER_REVISION_VALUE or similar.